### PR TITLE
Edited

### PR DESCRIPTION
**Pull request by Paryax**
- [DEMO LINK](https://Paryax.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://Paryax.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

**Review by kreisberg — APPROVED** [review]

File: src/index.html [hunk 1]
```diff
@@ -1,13 +1,55 @@
...
+  </head>
+  <body>
+    <section class="card" data-qa="card">
+      <img
+        src="./images/imac.jpeg"
+        alt="imacIcon"
+        class="card__icon"
+      >
+
+
```
Comment (ivan-kadykalo) [inline src/index.html:20]: No more than one empty line

File: src/index.html [hunk 2]
```diff
@@ -1,13 +1,55 @@
...
+      <h3 class="card__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h3>
+      <p class="card__product">
+        Product code: 195434
+      </p>
+
```
Comment (ivan-kadykalo) [inline src/index.html:27]: THe same

File: src/index.html [hunk 3]
```diff
@@ -1,13 +1,55 @@
...
+        class="card__icon"
+      >
+
+
+      <h3 class="card__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h3>
+      <p class="card__product">
+        Product code: 195434
+      </p>
```
Comment (ivan-kadykalo) [inline src/index.html:26]: Code style

File: src/index.html [hunk 4]
```diff
@@ -1,13 +1,55 @@
...
+
+
+      <ul class="stars stars--4">
+        <li class="stars__star"></li>
+        <li class="stars__star"></li>
+        <li class="stars__star"></li>
+        <li class="stars__star"></li>
+        <li class="stars__star"></li>
+        <p class="stars__review">Reviews: 5</p>
+      </ul>
```
Comment (ivan-kadykalo) [inline src/index.html:36]:
```suggestion
      <div class="some class name....">
	      <ul class="stars stars--4">
	        <li class="stars__star"></li>
	        <li class="stars__star"></li>
	        <li class="stars__star"></li>
	        <li class="stars__star"></li>
	        <li class="stars__star"></li>
	      </ul>
	      
      	<p class="stars__review">Reviews: 5</p>
      </div>
```

File: src/index.html [hunk 5]
```diff
@@ -1,13 +1,55 @@
...
+
+      <div class="card__link">
+        <a
+          class="card__item"
+          href="#buy"
+          data-qa="hover"
+        >
+          Buy
+        </a>
+      </div>
```
Comment (ivan-kadykalo) [inline src/index.html:51]:
```suggestion
        <a
          class="card__item"
          href="#buy"
          data-qa="hover"
        >
          Buy
        </a>
```

File: src/styles/stars.css [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+  background-repeat: no-repeat;
+  background-image: url(/images/star.svg);
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--1 .stars__star:first-child,
+.stars--2 .stars__star:nth-child(-n+2),
```
Comment (ivan-kadykalo) [inline src/styles/stars.css:26]: And so on
```suggestion
.stars--1 :first-child,
.stars--2 :nth-child(-n+2),
```

File: src/styles/card.css [hunk 7]
```diff
@@ -0,0 +1,95 @@
+.card {
+  width: 200px;
+  padding: 0 16px;
+
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__icon {
```
Comment (ivan-kadykalo) [inline src/styles/card.css:10]: Actually, it is an icon, it is image.

File: src/index.html [hunk 8]
```diff
@@ -1,13 +1,55 @@
...
+        src="./images/imac.jpeg"
+        alt="imacIcon"
+        class="card__icon"
+      >
+
+
+      <h3 class="card__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h3>
+      <p class="card__product">
```
Comment (ivan-kadykalo) [inline src/index.html:24]:
```suggestion
      <p class="card__code">
```

File: src/index.html [hunk 9]
```diff
@@ -1,13 +1,55 @@
...
+        <li class="stars__star"></li>
+        <li class="stars__star"></li>
+        <li class="stars__star"></li>
+        <li class="stars__star"></li>
+        <li class="stars__star"></li>
+        <p class="stars__review">Reviews: 5</p>
+      </ul>
+
+      <div class="card__price">
+        <p class="card__word">Price:</p>
```
Comment (ivan-kadykalo) [inline src/index.html:39]:
```suggestion
        <p class="card__price-title">Price:</p>
```

File: src/index.html [hunk 10]
```diff
@@ -1,13 +1,55 @@
...
+        <p class="card__digit">$2,199</p>
+      </div>
+
+      <div class="card__link">
+        <a
+          class="card__item"
```
Comment (ivan-kadykalo) [inline src/index.html:45]:
```suggestion
          class="card__button"
```

File: src/styles/card.css [hunk 11]
```diff
@@ -0,0 +1,95 @@
...
+  justify-content: center;
+
+  text-transform: uppercase;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  color: #fff;
+  text-decoration: none;
+
+  padding: 11px 69px 11px 68px;
```
Comment (ivan-kadykalo) [inline src/styles/card.css:84]: 😱
what will be if I will need to change the size of the card or button? I need to calc all this data again?

File: src/styles/card.css [hunk 12]
```diff
@@ -0,0 +1,95 @@
+.card {
+  width: 200px;
+  padding: 0 16px;
```
Comment (ivan-kadykalo) [inline src/styles/card.css:3]: You can set up paddings for the whole card, why 0?

**Review by ivan-kadykalo — CHANGES_REQUESTED** [review]
Good start👍

File: src/styles/card.css (hunk 11 above)
Comment (Paryax) [reply src/styles/card.css:84]: Everything stretches automatically. I tried it and it looks good

File: src/index.html (hunk 9 above)
Comment (solaryasha) [reply src/index.html:35]: Why don't you fix @ivan-kadykalo comments?

File: src/styles/card.css (hunk 12 above)
Comment (solaryasha) [reply src/styles/card.css:3]: Why don't you fix @ivan-kadykalo comments?

File: src/styles/card.css (hunk 11 above)
Comment (solaryasha) [reply src/styles/card.css:85]: Let's set the height of the button. Also, button's width should be 100% percent of the parent container

**Review by solaryasha — CHANGES_REQUESTED** [review]

**Review by ivan-kadykalo — CHANGES_REQUESTED** [review]
Why didn't you fix my comments?

**Review by ivan-kadykalo — CHANGES_REQUESTED** [review]
You still didn't fix my comments!!!
It is not normal.

**Pull request closed without merging**